{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"`cy.visit()` must be called with a `url` or an `options` object containing a `url` as its 1st argument\n                \n                https://on.cypress.io/visit\n                \n                Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `example to-do app`","children":[{"name":"displays two todo items by default","uid":"eff498b04c3ad9fe","parentUid":"f61cf463562c5a0ebd9034fab71a737e","status":"broken","time":{"start":1666187455553,"stop":1666187455735,"duration":182},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"f61cf463562c5a0ebd9034fab71a737e"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}